images/ose-azure-cloud-controller-manager.yml

```diff
@@ -0,0 +1,34 @@
+container_yaml:
+  go:
+    modules:
+    - module: github.com/openshift/cloud-provider-azure
+content:
+  source:
+    dockerfile: cloud-node-manager.Dockerfile
```

Comment:
This needs to also point to `cloud-controller-manager.Dockerfile` for a full Azure support.